generators/base/generator-base-private.mjs

```diff
@@ -183,13 +175,10 @@ export default class PrivateBase extends Generator {
    * @returns {boolean} true if success; false otherwise
    */
   gitMove(source, dest) {
```

Comment:
We probably should remove gitMove.
It was used to rename i18n files that change only the casing, breaking case insensitive filesystem.
Add an anonymous function at https://github.com/jhipster/generator-jhipster/blob/c4223686a9457e99c50a41584b950f3cafa8e23c/generators/cleanup.mjs#L362.
So we can decide if we can drop this upgrade logic later.

Reply:
I guess that there are a lot of file move (for example during jhipster upgrade).
We'll choose the best one once every filesystem move will be centralized.

Reply:
We don't do git operations at main generators.
We could do at upgrade generator, move there then.

Reply:
Sure, fully agree! Just want to be sure that the gitMove is only used this way.
 
Would you agree with this procedure:
1. extract the logic in the relevant's blueprint 'logic' folder in a serie of PR
2. check the usage of this business logic and move the generator method in the respective generator in another serie of PR
3. check if there is value exposing the generator method and remove it from the API if there isn't.

This approach looks straightforward, dumb enough to be reviewed quickly (and it should be because there will be a LOT of PRs to achieve before the 8th release) and less error prone than mixing purposes

Would it be ok to proceed this way?
I'll create the 3 different GH issue and reference the according PRs

Reply:
Done sir

Reply:
rollbacked for further investigation later on